unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/DefaultChatEntry.cs

```diff
@@ -361,9 +364,12 @@ private void CheckHoverCoordinates()
 
         isOverCoordinates = true;
         TMP_LinkInfo linkInfo = body.textInfo.linkInfo[linkIndex];
-        currentCoordinates = CoordinateUtils.ParseCoordinatesString(linkInfo.GetLinkID().ToString());
-        hoverGotoPanelTimer = timeToHoverGotoPanel;
-        OnCancelHover?.Invoke();
+        if (!ChannelUtils.IsAChannel(linkInfo.GetLinkID()))
```

Comment:
i think the condition here should be: `CoordinateUtils.IsCoordinate(...)`

Reply:
No, it is intentional. In this case, in order to avoid conflicts between coordinate/channels links detection, we ensure the text is not a channel link.

It will change when we move the coordinate links detection logic from the chat entry to an independent component as we are already doing with channels.